.editorconfig

```diff
@@ -0,0 +1,21 @@
+# This file is for unifying the coding style for different editors and IDEs
```

Comment:
I am using remark lint (the tool that codacy uses for linting) to format the markdown.

@pauloribeiro-codacy is putting a configuration for this in this other branch https://github.com/codacy/chart/pull/293/files

Reply:
This is a universal file that most editors pick if there is no other tool

Reply:
As it is now, I don't think there will be a collision between this configuration file and the one for the linter. If there is, I will adapt the `.remarkrc` file.